src/app/topbar/topbar-container.component.ts

```diff
@@ -1,24 +1,65 @@
 import { Component, HostBinding } from '@angular/core';
+import { JwtHelper } from 'angular2-jwt';
+import { AuthService } from '../services/auth.service';
 
 @Component({
   selector: 'app-topbar-container',
   template: `
     <nav class="nav-left">
-      <a *ngIf="apiTokenEnabled" class="btn btn-default">Set API Token</a>
-      <a *ngIf="loginEnabled" class="login-link">Login</a>
+      <input type="text"
```

Comment:
I think we're going to need to handle refresh tokens rather than the actual API tokens. Users would probably expect to be able to get a token from the app and use it here. Our python client takes a refresh token as well (it can also handle an api token) so I think from a consistency and usability standpoint it would be better.

From a re-usability standpoint (R+D related), we probably want to make this as modular as possible. I think that someone customizing the `auth.service` by implementing a few standard methods would make sense.